navigator_engine/api.py

```diff
@@ -78,4 +78,8 @@ def action():
         }
     ```
     """
-    return decide()
+    result = decide().json
+    action = result['decision']
+    action['currentMilestone'] = result['progress']['currentMilestoneID']
+    action['skipped'] = result['decision']['id'] in result['skippedActions']
+    return jsonify(action)
```

Comment:
I agree that under the current design this would be fine.  Though we did have the original design idea  that you should be able to get the action details without rerunning the entire decide process.   This action end point should be a super fast endpoint compared to the decide endpoint.  The fact we are running the decide process behind the action endpoint is just a hack for now?